blendergltf.py

```diff
@@ -371,6 +371,37 @@ def export_material(material):
 
 
 def export_meshes(meshes, skinned_meshes):
+    def triangulate(indices):
+
+        # Triangulate each polygon if necessary
+        if len(indices) == 3:
+            # No triangulation necessary
+            return indices
+        elif len(indices) > 3:
+            # Triangulation necessary
+            total_indices = []
+            for i in range(len(indices) - 2):
+                total_indices += (indices[-1], indices[i], indices[i + 1])
+            return total_indices
+        else:
+            # Bad polygon, probably an edge or something strange.
+            raise RuntimeError(
+                "Invalid polygon with {} vertices.".format(len(indices))
+            )
+
+    def get_index_type_stride(max_index):
+        if max_index > 65535:
+            # Too large! In the future we may want to just split the mesh into
+            # smaller parts.
+            raise RuntimeError(
```

Comment:
I think this would be better as a warning (like we do with lights). That way the rest of the scene can still be exported, but the mesh will be empty/missing.